components/drawer/events.md

```diff
@@ -131,3 +133,71 @@ The `ExpandedChanged` event fires every time the component's state is changed -
 @[template](/_contentTemplates/common/general-info.md#event-callback-can-be-async)
 
 @[template](/_contentTemplates/common/issues-and-warnings.md#valuechanged-lambda-required)
+
+## OnItemRender
+
+The OnItemRender event fires when each item in the Drawer renders. It allows you to customize the appearance of an item. Note that the event does not work alongside a [`<Template>`]({%slug drawer-templates%}#template), as the template disables all built-in features of the Drawer.
+
+The event handler receives as an argument an object of type `DrawerItemRenderEventArgs` that contains: 
+
+| Property | Type | Description |
+| --- | --- | --- |
+| `Item`   | `object` |The current item that renders in the Drawer. |
+| `Class`  | `string` |The custom CSS class that will be added to the item. |
+
+>caption The example shows you how to Customize the appearance of the Drawer items depending on the Drawer Mode.
```

Comment:
Structure the captions as headings instead of complete sentences.
```suggestion
>caption Customizing the appearance of the Drawer items based on the Drawer mode.
```